components/Mitra/index.vue

```diff
@@ -181,18 +185,20 @@ export default {
     }
 
     &-status {
-      @apply;
       &-green {
-        @apply px-4 py-[6px] rounded-[31px] bg-green-50 border-green-400;
-      }
-      &-blue {
-        @apply px-4 py-[6px] rounded-[31px] bg-blue-50 border-blue-400;
+        @apply px-4 py-[6px] rounded-[31px] bg-green-50 border border-green-400;
       }
       &-yellow {
-        @apply px-4 py-[6px] rounded-[31px] bg-yellow-50 border-yellow-400;
+        @apply px-4 py-[6px] rounded-[31px] bg-yellow-50 border border-yellow-400;
       }
       &-red {
-        @apply px-4 py-[6px] rounded-[31px] bg-red-50 border-red-400;
+        @apply px-4 py-[6px] rounded-[31px] bg-red-50 border border-red-400;
+      }
+      &-blue {
+        @apply px-4 py-[6px] rounded-[31px] bg-blue-50 border border-blue-400;
+      }
+      &-gray {
+        @apply px-4 py-[6px] rounded-[31px] bg-gray-200 border border-gray-400;
```

Comment:
Suggestion:

Hello, kak @yoslie. 
Since we use BEM Convention for the naming convention. We can write like this 🚀 

 ```html
         <span
            :class="{
              'mitra__datatable-status' : true,
              'mitra__datatable-status--green' : item.status_partner === 'Terverifikasi',
              'mitra__datatable-status--yellow' : item.status_partner === 'Menunggu Verifikasi',
              'mitra__datatable-status--red' : item.status_partner === 'Ditolak',
              'mitra__datatable-status--blue' : item.status_partner === 'Aktif',
              'mitra__datatable-status--gray' : item.status_partner === 'Nonaktif',
            }"
          >
            {{ item.status_partner }}
          </span>
``` 

```css
&-status {
      @apply px-4 py-[6px] rounded-[31px] border;

      &--green {
        @apply bg-green-50 border-green-400;
      }
      &--yellow {
        @apply bg-yellow-50 border-yellow-400;
      }
      &--red {
        @apply bg-red-50 border-red-400;
      }
      &--blue {
        @apply bg-blue-50 border-blue-400;
      }
      &--gray {
        @apply bg-gray-200 border-gray-400;
      }
    }
```

Reply:
sure thanks for advice, changes [here](https://github.com/jabardigitalservice/dedi-cms/pull/60/commits/ee495d77ceb4e0dc29c635bca0a9d3d878a44dda)